src/Traits/HasImages.php

```diff
@@ -55,7 +58,7 @@ protected function buildCsvImages(CSVConfig $csvConfig): string
         if (array_key_exists('', $this->images)) {
             $images = $this->images[''];
 
-            for ($i = 0; $i < $csvConfig->getImageCount(); $i++) {
+            for ($i = 0; $i < $csvConfig->getImageCount(); ++$i) {
```

Comment:
why did this change ?

Reply:
https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#postincdectopreincdecrector

There is not really an explanation here, so we can revert it